frontend/src/app/components/blocks-list/blocks-list.component.html

```diff
@@ -123,8 +123,8 @@ <h1 *ngIf="!widget" class="float-left" i18n="master-page.blocks">Blocks</h1>
       (pageChange)="pageChange(page)" [boundaryLinks]="true" [ellipses]="false">
     </ngb-pagination>
 
-    <div class="clearfix"></div>
-    <br>
+    <div *ngIf="!widget" class="clearfix"></div>
```

Comment:
If you wrap both elements in `<ng-template [ngIf]="!widget">` you save the logic two if statements

Reply:
Fixed